File: mobile/src/main/java/com/bradcypert/textico/services/HeadlessSmsSendService.java
package com.bradcypert.textico.services;

import android.app.Service;
import android.content.Intent;
import android.os.IBinder;
import android.support.annotation.Nullable;

/**
 * Created by bradc on 3/13/2017.
 */

public class HeadlessSmsSendService extends Service {
    @Nullable
    @Override
    public IBinder onBind(Intent intent) {
        return null;
    }
}

File: mobile/src/main/java/com/bradcypert/textico/services/HeadlessSmsSendService.kt
package com.bradcypert.textico.services

import android.app.Service
import android.content.Intent
import android.os.IBinder

/**
 * Created by bradc on 3/13/2017.
 */

class HeadlessSmsSendService : Service() {
    override fun onBind(intent: Intent): IBinder? {
        return null
    }
}
